dev-docs/bidders/janet.md

```diff
@@ -13,7 +13,7 @@ usp_supported: true
 safeframes_ok: true
 prebid_member: true
 pbjs: true
-pbs: false
+pbs: true
```

Comment:
There's also a new flag for your bidder documentation introduced in #3694

`multiformat_supported` . Possible values are

* `will-bid-on-any` - bidder will bid on any media type
* `will-bid-on-one` - bidder will prioritize one media type and ignores the others
* `will-not-bid` - bidder will not bid at all

This helps publishers / sales houses to know up front how adapter will behave in a multi-format environment

Reply:
@ChrisHuie Thank you very much, Chris
@muuki88 Thank you for the hint. We will update our documentation accordingly